### PR TITLE
Spectral

### PR DESCRIPTION
**Pull request by cgiger00**
also has the LCM structs defined at the esw/teleop interface meeting last wednesday. technically has never been tested with all 4 sensors hooked up, but works with the triad hooked up along w one single, so should work for all 4.

Probably wont pass travis because of IOError...when it's connected to the bb this error won't be thrown

### CODE REVIEW

File: rover_msgs/FootCmd.lcm [hunk 1]
```diff
@@ -4,3 +4,4 @@ struct FootCmd {
 	double sensor; // [-1.0, 1.0]
     double claw; // [-1.0, 1.0]
 }
+
```
Comment (raytitan) [inline rover_msgs/FootCmd.lcm:7]: could you revert this?

File: rover_msgs/IMUData.lcm [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    double gyro_y_dps;
+    double gyro_z_dps;
+    double mag_x_uT;
+    double mag_y_uT;
+    double mag_z_uT;
+    double roll_rad;
+    double pitch_rad;
+    double yaw_rad;
+    double bearing_deg;
+}
```
Comment (raytitan) [inline rover_msgs/IMUData.lcm:17]: This should definitely not be in this pull request

File: rover_msgs/MosfetCmd.lcm [hunk 3]
```diff
@@ -1,17 +1,12 @@
...
-	const int8_t DEV9 = 9;
+	const int8_t HEAT_W_12V 		= 0;
+	const int8_t UV_LED_REP_5V 		= 1;
+	const int8_t LED_24V 			= 2;
+	const int8_t HEAT_B_12V 		= 3;
+	const int8_t HEAT_Y_12V 		= 4;
+	const int8_t RA_LASER_3_3V 		= 5;
 
     int8_t device;
     boolean enable;
```
Comment (raytitan) [inline rover_msgs/MosfetCmd.lcm:12]: I'd rather this be a [spectral] PR and then you can have a seperate [mosfet] PR or something

File: rover_msgs/ServoCmd.lcm [hunk 4]
```diff
@@ -1,6 +1,6 @@
 package rover_msgs;
 
 struct ServoCmd {
-	string id;
+	string id; //<test>_<box color>
 	double position; //degrees
 }
```
Comment (raytitan) [inline rover_msgs/ServoCmd.lcm:6]: This should be in your servo PR

File: rover_msgs/ThermistorCmd.lcm [hunk 5]
```diff
@@ -0,0 +1,6 @@
+package rover_msgs;
+
+struct ThermistorCmd {
+	double temperature_C; // Celsius
+	string thermistor; // Either thermistor white, blue, or yellow
+}
```
Comment (raytitan) [inline rover_msgs/ThermistorCmd.lcm:6]: This should not be in this PR

**Comment by cgiger00** [comment]
updated spectral ports

**Pull request merged**